nectar/src/config/settings.rs

```diff
@@ -163,10 +163,14 @@ pub struct Maker {
     /// Spread to apply to the mid-market rate, format is permyriad. E.g. 5.20
     /// is 5.2% spread
     pub spread: Spread,
+    // TODO: Probably move that under something common to fee_strategies.
+    // I did not realize I would still need it.
```

Comment:
```suggestion
```

Maybe remove the comment above as well?

Reply:
No, I want to clean that up as part the follow-up PR I am suggesting.

@thomaseizinger FYI, if you end up touching the settings before me we may want to re-think what should go where: fee strategy, max possible fee. Under maker? together?